NamVr/DiscordBot-Template
# Better help slash command + some other improvements

See #11 

See commits for more details, but here is the summary:
- Improve the `help` slash command so users can get help on a single command
- Update Discord.js to latest version
- Add the LICENSE file with detailed license and so the link in readme is no longer broken
- Update some dependencies for security (audits)

I did some testing so it should be safe to implement this.

Happy to contribute to this project, it helped me a lot to learn about the new stuff of Discord.js v13!
